lettuce/django/management/commands/harvest.py

```diff
@@ -36,7 +36,7 @@
 class Command(BaseCommand):
     help = u'Run lettuce tests all along installed apps'
     args = '[PATH to feature file or folder]'
-    requires_model_validation = False
+    requires_model_validation = requires_system_checks = False
```

Comment:
This change breaks the harvest command - see https://github.com/gabrielfalcao/lettuce/pull/525